environment.md

```diff
@@ -1,19 +1,35 @@
-# Environment
+# Server Environment
 
-WordPress will run in a minimum server environment. However, WordPress doesn't run optimally on the minimum system requirements. This section will cover the _recommended_ server environment for WordPress to run more effectively.
+Although WordPress can work in almost any environment, even very minimal ones, it must be acknowledged that it does not work completely well in these. That's why here we are going to make some minimum recommendations of the environment in which it would work most effectively.
 
-## Recommended Servers
+## Web Server
 
-WordPress recommends using one of these servers:
+The web server is the system where the files of the web site are hosted and where the users arrive to consult them. There are many web servers, and in principle anyone who supports a PHP connection should be able to work with WordPress.
 
-*   Apache 2.4 or higher
-*   NGINX 1.14 or higher
+When it comes to the server, web, WordPress works best with these (sorted alphabetically):
+
+* [Apache HTTPD](https://httpd.apache.org/) 2.4.x
+* [LiteSpeed Web Server](https://www.litespeedtech.com/products/litespeed-web-server) 5.3.x / 5.4.x
+* [nginx](https://nginx.org/) 1.18.x / 1.17x
++ [OpenLiteSpeed](https://openlitespeed.org/) 1.7.x / 1.6.x / 1.5.x / 1.4.x
+
+Remember that if you have a website running in production, it is recommended to use the latest stable version of each of the web servers (mainly for security, rather than functionality), but not alpha, beta or candidate (RC) versions.
 
 ## PHP
 
-### PHP Version
+PHP is a programming language on which WordPress code is based. This language runs on the server and it is important to keep it up to date, both for security and mainly for functionality.
+
+WordPress supports many versions of PHP, some already obsolete, but always all that are supported and maintained.
+
+Officially the WordPress core supports from PHP 7.0 to PHP 7.4. However, not all themes or plugins are supported.
+
+When it comes to PHP, WordPress (including its extensions) works best with the following versions:
```

Comment:
Suggest removing "including its extensions," since this means plugins and themes to me, and their compatibility varies quite a bit (as is mentioned above).

Reply:
Changed to:

When it comes to PHP, WordPress works best with the following versions: